px-demo-snippet.html

```diff
@@ -77,7 +79,7 @@
 <link rel="import" href="css/px-demo-snippet-styles.html">
 <dom-module id="px-demo-snippet">
   <template>
-    <style include="px-demo-snippet-styles"></style>
+    <style include="px-demo-snippet-styles" is="custom-style"></style>
```

Comment:
take is="custom-style" off. not needed in a host element root, also deprecated in Polymer 2. it won't do anything different here if you remove it.